docs/docs/monitoring/analytics/getting-started-with-analytics.mdx

```diff
@@ -61,6 +61,41 @@ The pipeline will compute analytics as soon as the docker container
 is successfully deployed and connected to your data warehouse and
 Kafka instances.
 
+# Types of metrics
+
+Metrics collected from your assistant can broadly be categorized as
+
+- User Analytics: Who are the users of the assistant, and how do they feel 
+about it?  Examples: demographics, channels, sentiment analysis
+- Usage Analytics:  How is the assistant’s overall health and what kind of 
+traffic is coming to it?  Examples:  total number of sessions, time per 
+session, errors and error rates
+- Conversation Analytics: What happened during the conversation?  
+Examples: number of messages sent, abandonment depth, number of topics 
+introduced by user, top N intents
+- Business Analytics: How is the assistant performing with regard to business goals?
+Examples: ROI of assistant per LoB, time comparison of assistant vs agent, containment rate
+
+In this version of the Analytics pipeline, measurement of the following metrics 
+is possible 
```

Comment:
I'm not 100% sure about listing all the metrics below. Because 1) this list is probably incomplete (users get creative!) and 2) this is going to be hard to keep up to date (maybe). But up to you :)